<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidgetDemoClass</class>
 <widget class="QWidget" name="TabWidgetDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidgetDemo</string>
  </property>
  <widget class="QPushButton" name="pushButtonDeleteLine">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRemakeItem">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改Item</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInsertLine">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>老师名字</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>课程名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>上课时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>课后读物</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>蓝老师</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>一阳指</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>20:00-22:00</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>金瓶梅</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TabWidgetDemo.qrc"/>
 </resources>
 <connections/>
</ui>
